### DISCUSSION TITLE
Example of how to change the website language

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [withastro/starlight#1439] [Q&A] [answered]
Hello, is there a way to change the language of the search magnifying glass and the head or "on this page > overview"

**Comment by HiDeoo** [comment, marked as answer]
Hey :wave:

Starlight UI strings are by default available, at the time of writing, in Arabic, Chinese, Czech, Danish, Dutch, English, French, Galician, German, Hebrew, Hindi, Indonesian, Italian, Japanese, Korean, Norwegian Bokmål, Persian, Portuguese, Romanian, Russian, Spanish, Swedish, Turkish, Ukrainian, and Vietnamese by default ([reference](https://starlight.astro.build/guides/i18n/#translate-starlights-ui) to the up-to-date list).

If this is one of the languages you want to use and your site is mono-lingual, you can follow this [guide](https://starlight.astro.build/guides/i18n/#monolingual-sites) to setup a single language site in one of these languages. You will need to set the desired language as the `root` in your `locales` config (in your `astro.config.mjs` file).

For example, in French, this would be:

```js
import { defineConfig } from 'astro/config';
import starlight from '@astrojs/starlight';

export default defineConfig({
  integrations: [
    starlight({
      title: 'Ma documentation',
      locales: {
        root: {
          label: 'Français',
          lang: 'fr',
        },
      },
    }),
  ],
});
```

If your website is multilingual, Starlight provides built-in support for this and can be setup by following this [guide](https://starlight.astro.build/guides/i18n/#configure-i18n).

If the language you desire is not listed above, you can follow this [guide](https://starlight.astro.build/guides/i18n/#translate-starlights-ui) to add a custom language by providing your own translations.
We also welcome [contributions to add more default languages](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md#translating-starlights-ui).

If you have any follow up questions, feel free to ask.

**Reply by sebastianjnuwu** [reply]
Thank you for your help! I found what I needed ^^
